examples/scripts/run_simulation.py

```diff
@@ -0,0 +1,6 @@
+import pybamm
+
+model = pybamm.lithium_ion.SPM()
+sim = pybamm.Simulation(model)
+sim.solve()
+sim.plot()
```

Comment:
Could you add this code snippet to the readme as a super quick way of running a model? e.g. (DFN is more standard than SPM)

## How do I use PyBaMM?

The easiest way to use PyBaMM is to run a 1A constant-current discharge with a model of your choice with all the default settings:
```python3
import pybamm
model = pybamm.lithium_ion.DFN() # Doyle-Fuller-Newman model
sim = pybamm.Simulation(model)
sim.solve()
sim.plot()
```
However, much greater customisation is available. It is possible to change the physics, parameter values, geometry, submesh type,  number of submesh points, methods for spatial discretisation and solver for integration (see DFN [script](examples/scripts/DFN.py) or [notebook](examples/notebooks/models/dfn.ipynb)).

Further details on how to customise simulations and add your own models and settings are provided in the [examples and documentation](#examples-and-documentation).

Reply:
1A or 1C here?

Reply:
Changed to 1C